cephprimarystorage/cephprimarystorage/cephdriver.py

```
@@ -40,24 +40,23 @@ def create_volume(self, cmd, rsp, agent=None):
         if cmd.skipIfExisting and shell.run("rbd info %s" % path) == 0:
             return rsp
 
-        if ceph.is_xsky():
-            # do NOT round to MB
+        multiple_of_MB = cmd.size % sizeunit.m == 0
+        if cmd.format == 'qcow2':
+            vol_path = "rbd:%s" % path
+            linux.qcow2_create(vol_path, cmd.size, False)
```

Ruff (0.8.2)

`linux` may be undefined, or defined from star imports

(F405)